tests/test_handler_rocauc.py

```diff
@@ -34,6 +34,8 @@ def test_compute(self):
         y = torch.Tensor([[0], [1]])
         y_pred = act(y_pred)
         y = to_onehot(y)
+        # test a list of channel-first tensors
+        y_pred, y = list(y_pred), list(y)
         auc_metric.update([y_pred, y])
```

Comment:
In `monai/handlers/roc_auc.py`, the type hint for `update` is:
`def update(self, output: Tuple[torch.Tensor, torch.Tensor]`, here the input is `List[torch.Tensor, torch.Tensor]`, should the type be changed into `Union[Tuple[torch.Tensor, torch.Tensor], List[torch.Tensor, torch.Tensor]]` ?

Reply:
`List[torch.Tensor, torch.Tensor]` will complain about too many arguments since it's used to definite an arbitrary length list of items, so `List[T]` is a list of zero or more `T`. For `Tuple[T,T]` this states a tuple of precisely two `T` objects.

Reply:
Hi @yiheng-wang-nv ,

I used `output: Tuple[torch.Tensor, torch.Tensor]` because ignite defined the API and args. It actually can support tuple and list.

Thanks.